### ISSUE TITLE
East side vendor in Por's Pub should be a banker

### ISSUE THREAD

**Issue by stormhalter** [jkachhad/Stormhalter#1459]
[Suggested by stardust]

---
[View on Discord](https://discord.com/channels/1056432310632841276/1530608177798975508/1530608177798975508)

---
[View on Discord](https://discord.com/channels/577503035526348810/1532140708436119693/1532140708436119693)

**Comment by Chilly021** [comment]
Agreed purely on Lands implementation though I think @jkachhad  would need to give feedback here, due to we added a bank ring to circumvent adding more banks etc. 

![image](https://user-images.githubusercontent.com/90510109/206710573-1ded5009-13be-4874-9062-7a0bddfc699e.png)
